network-api/networkapi/mozfest/migrations/0017_update_wide_images.py

```diff
@@ -0,0 +1,65 @@
+# # Generated by Django 3.0.14 on 2021-08-10 18:19
```

Comment:
This file loops through pages in the mozfest app, checks for any images that are set to "wide" and then updates the radio select field to be wide as well.